superset/assets/javascripts/locales.jsx

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable global-require, import/no-dynamic-require */
+import Jed from 'jed';
+import React from 'react';
+import { sprintf } from 'sprintf-js';
+
+let i18n = null;
+let i18nPromise = null;
+
+function formatForReact(formatString, args) {
```

Comment:
I don't understand what this function does it appears to me that `args` can be React elements, but I don't understand the use case.